exhibitor/models.py

```diff
@@ -19,10 +19,36 @@ def __str__(self):
         return self.username
 
 class Moc(models.Model):
+    CASTLE = 'Castle/Starożytność'
+    ARCHITEKTURA = 'Architektura'
+    KOMIKS = 'Komiks/Anime/Manga'
+    MILITARIA = 'Militaria'
+    POJAZDY_MINIFIG = 'Pojazdy Minifig'
+    POJAZDY_TECHNIC = 'Pojazdy technic'
+    SPACE = 'Space/Sci-fi'
+    KOLEJNICTWO = 'Kolejnictwo'
+    CATEGORY = [
+        ('Castle/Starożytność', 'Castle/Starożytność'),
+        ('Architektura', 'Architektura'),
+        ('Komiks/Anime/Manga', 'Komiks/Anime/Manga'),
+        ('Militaria', 'Militaria'),
+        ('Pojazdy Minifig', 'Pojazdy Minifig'),
+        ('Pojazdy technic', 'Pojazdy technic'),
+        ('Space/Sci-fi', 'Space/Sci-fi'),
+        ('Kolejnictwo', 'Kolejnictwo'),
+    ]
+    FIRST_SIZE = 0.5
+    SECOND_SIZE = 1
+    THIRD_SIZE = 2
+    SIZE = [
```

Comment:
this is exactly what I meant in the previous PR 👯 

I see also this PR contains changes from previous one so I am leaving up to you how you resolve this. My comment about keeping one migration still valid. It's not big deal now but for learning purpose you should be able to manage migrations.